app/controllers/task_controller.rb

```diff
@@ -12,14 +12,14 @@ def show
   end
 
   def new
-    @new_task = Task.new
+    @task = Task.new
 
     render :add_task
```

Comment:
It would be good to stick with the matching action and view names (new & new) - especially for these most basic actions. Same recommendation for the edit action.